policyengine_us/parameters/gov/states/wa/tax/income/capital_gains/rate.yaml

```diff
@@ -1,12 +1,12 @@
 description: Washington capital gains tax rate.
 metadata:
   label: Washington capital gains tax rate
-  name: wa_ltcg_rate
   unit: /1
   period: year
   reference:
-    - title: "RCW 82.87.040(1) Tax imposed—Long-term capital assets."
+    - title: RCW 82.87.040(1) Tax imposed—Long-term capital assets.
       href: https://app.leg.wa.gov/RCW/default.aspx?cite=82.87.040
+    - title: Capital gains tax | Washington Department of Revenue
+      href: https://dor.wa.gov/taxes-rates/other-taxes/capital-gains-tax#Background
```

Comment:
![image](https://github.com/user-attachments/assets/cab4e1c4-1bd9-409e-8ac0-e371dface3af)

Similar language should be present for all the values that are adjusted annually